src/routes/plans/+page.svelte

```diff
@@ -493,6 +493,7 @@
     }
   }
 
+  // TODO: logging reveals that this doesn't get called! As such, usage of on:deleteItem is not done for source grid in ExternalSourceManager. We should probably remove this and the on:deleteItem too to reduce confusion
```

Comment:
Is this still true or a TODO?

Reply:
This is used when the user right clicks on the plans table and deletes a plan via context menu